buildSrc/build.gradle.kts

```diff
@@ -1,17 +1,13 @@
 plugins {
-    groovy
     `kotlin-dsl`
 }
 
 repositories {
-    mavenCentral()
     gradlePluginPortal()
-    // until Dokka 1.8.0 is released and we no longer need dev builds, see https://github.com/kordlib/kord/pull/755
-    maven("https://maven.pkg.jetbrains.space/kotlin/p/dokka/dev")
+    mavenCentral()
+    maven("https://europe-west3-maven.pkg.dev/mik-music/kord")
```

Comment:
It's what I used for the Gradle Plugin, since we can't submit it to the Gradle Plugin Portal, as it's not designed for public use, and I don't really want to put it on Sonatype either, since I don't really want it to be discoverable, as it isn't designed to be used outside Kord

Reply:
ok 👍

Reply:
maybe a comment about that would be good then